### PR TITLE
Do not check if any archive is running when a segment is forced

### PR DESCRIPTION
**Pull request by tsteur**
From a support request. eg user has

```crontab
6 * * * * * console core:archive --force-idsegment=1
6 * * * * * console core:archive --force-idsegment=2
6 * * * * * console core:archive --force-idsegment=3
```

Then currently it would skip some segments because it checks if any archive is already running. After the launching the first one for segment 1, an archive is already running and would probably never launch 2nd and 3rd segment AFAIK.

@mattab

### CODE REVIEW

**Review by sgiehl — APPROVED** [review]

**Comment by tsteur** [comment]
@mattab fyi waiting for your confirmation that it fixes the user's issue before merging

**Pull request merged**
